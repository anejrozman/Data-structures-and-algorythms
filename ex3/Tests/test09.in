17,61,3
9.832196917360491,6.280296671746237,1.903705244909897,4.853336478215307,6.129528047364767,4.060389094060168,4.636124335522108,8.393917071871607,9.921013454828818,9.271149407443996,4.279255649001746,1.0857555102881056,9.808570392870134,3.540144442003593,3.452373985262221,2.645036469498158,3.9321208237057887
7,1,513.464
10,11,517.062
13,15,262.117
4,2,263.836
9,14,392.09
6,4,776.534
7,9,907.666
6,0,651.073
8,12,779.753
6,8,527.708
0,8,783.327
5,6,146.969
7,14,787.299
6,15,403.387
2,12,793.571
12,1,665.9
10,14,795.653
1,14,538.866
4,10,281.395
8,16,927.365
8,3,160.897
1,5,282.716
4,14,547.018
8,10,293.593
2,3,422.584
2,8,553.399
14,16,554.534
9,13,427.225
1,0,428.29
7,6,942.067
10,16,303.095
10,1,560.886
16,0,567.399
9,5,184.733
4,5,183.639
14,15,62.159
13,12,191.645
1,4,833.628
14,3,961.273
4,0,195.809
0,5,453.968
4,13,330.778
14,5,971.299
0,13,845.245
2,9,976.413
0,9,80.293
11,13,466.163
1,15,345.828
9,16,90.588
2,11,986.147
5,15,347.901
10,0,94.473
16,4,474.105
8,14,606.355
4,7,101.23
15,2,998.014
15,3,104.567
12,15,873.099
7,10,124.306
7,15,626.061
15,16,246.224
