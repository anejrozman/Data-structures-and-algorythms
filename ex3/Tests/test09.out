15,3447.0439
1,0
1,0
